CoreDistributedCache/NHibernate.Caches.CoreDistributedCache/CoreDistributedCache.cs

```diff
@@ -25,16 +25,115 @@
 using System.Collections.Generic;
 using System.IO;
 using System.Runtime.Serialization.Formatters.Binary;
+using System.Threading;
+using System.Threading.Tasks;
 using Microsoft.Extensions.Caching.Distributed;
 using NHibernate.Caches.Util;
 using NHibernate.Util;
 
 namespace NHibernate.Caches.CoreDistributedCache
 {
+	// 6.0 TODO: replace that class by its base
 	/// <summary>
 	/// Pluggable cache implementation using <see cref="IDistributedCache"/> implementations.
 	/// </summary>
-	public partial class CoreDistributedCache : ICache
+	public class CoreDistributedCache : CoreDistributedCacheBase,
+#pragma warning disable 618
+		ICache
+#pragma warning restore 618
```

Comment:
We do not need it for avoiding a binary breaking change, but it avoids having to rewrite all the XML comments.